specification/security/resource-manager/Microsoft.Security/stable/2024-08-01/standardAssignments.json

```diff
@@ -0,0 +1,440 @@
+              "format": "date-time",
+              "readOnly": true
+            },
+            "evidence": {
+              "type": "array",
+              "description": "Array of links to attestation evidence",
+              "items": {
+                "$ref": "#/definitions/AttestationEvidence"
+              }
+            }
+          }
+        },
+        "metadata": {
+          "$ref": "#/definitions/StandardAssignmentMetadata",
+          "description": "The standard assignment metadata."
+        }
+      }
+    },
+    "StandardAssignmentMetadata": {
+      "type": "object",
+      "description": "The standard assignment metadata",
+      "properties": {
+        "createdBy": {
+          "description": "standard assignment Created by object id (GUID)",
+          "type": "string",
+          "readOnly": true
+        },
+        "createdOn": {
+          "description": "standard assignment creation date",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "lastUpdatedBy": {
+          "description": "standard assignment last updated by object id (GUID)",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastUpdatedOn": {
+          "description": "standard assignment last update date",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        }
+      }
+    },
+    "AssignedStandardItem": {
+      "type": "object",
+      "description": "describe the properties of a of a standard assignments object reference",
+      "properties": {
+        "id": {
+          "description": "full resourceId of the Microsoft.Security/standard object",
+          "type": "string"
+        }
+      }
+    },
+    "AssignedAssessmentItem": {
+      "type": "object",
+      "description": "describe the properties of a security assessment object reference (by key)",
+      "properties": {
+        "assessmentKey": {
+          "description": "unique key to a security assessment object",
+          "type": "string"
+        }
+      }
+    },
+    "AttestationEvidence": {
+      "type": "object",
+      "description": "describe the properties of a assignment attestation",
+      "properties": {
+        "description": {
+          "description": "The description of the evidence",
+          "type": "string"
+        },
+        "sourceUrl": {
+          "description": "The source url of the evidence",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "StandardAssignmentName": {
+      "name": "standardAssignmentName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "pattern": "[{]?[0-9a-fA-F]{8}-(?:[0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}[}]?$",
+      "description": "The standard assignments assignment key - unique key for the standard assignment",
+      "x-ms-parameter-location": "method"
+    },
+    "StandardsAssignmentBody": {
```

Comment:
>StandardsAssignmentBody [](http://example.com/codeflow?start=5&length=23)

do we need this then ? StandardAssignment should be fine then ?

Reply:
@ramoka178 
This is the same
When you create the standard assignment, you give the body as a parameter
in the response, you can see that there are also the resource details (id. name, type and etc...) this is the norma that we are using in Micorosft.security